Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -163,46 +172,41 @@ namespace OpenInfraPlatform
 						dAngle += M_TWOPI;
 					}
 				}
-
-				bool areEqual(const double first,const double second) const
-				{
-					if (abs(first - second) < getPrecision()) 
-					{
-						return true;
-					}
-					return false;
-				}
-
-				bool areEqual(const carve::geom::vector<2>& first, const  carve::geom::vector<2>& second) const
+				/*!Verify two points of series whith precision

```

Comment:
What series are you talking about here? The function compares two doubles for equality to the specified precision.

Reply:
Resolved: b2f7040